.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 /composer.lock
 .phpunit.result.cache
+.idea
```

Comment:
This isn't the responsibility of the package. The IDE is the developer's choice, and any ignores should be in the developer's global _.gitignore_ (Usually as `/.idea/` as it's typically at the root of the repo).